tests/apps/study/test_study_api.py

```diff
@@ -56,3 +57,57 @@ def test_study_detail_api(db_keys):
         "url": "/study/1/",
         "rob_response_values": [27, 26, 25, 22, 24, 20],
     }
+
+
+@pytest.mark.django_db
+def test_study_create_api(db_keys):
```

Comment:
for long tests like this, I'm switching to use this pattern:

```
class TestStudyCreateApi:

    def test_permissions():
        ...
   def test_bad_requests():
        ...
   def test_valid_requests():
        ...
```

Breaks things up a little bit

Reply:
Good idea, fixed!